pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch_handlerchain.go

```diff
@@ -71,16 +76,26 @@ func BuildHandlerChain(genericConfig *genericapiserver.Config, kubeInformers inf
 		nil
 }
 
-func newWebConsoleProxy(kubeInformers informers.SharedInformerFactory, legacyServiceServingCertSignerCABundle string) (http.Handler, error) {
+func newWebConsoleProxy(kubeInformers informers.SharedInformerFactory, legacyServiceServingCertSignerCABundle string) (http.Handler, *serviceCABundleUpdater, error) {
```

Comment:
Change this, and update `newWebConsolePublicURLAccessor` to follow the pattern of `NewOAuthServerHandler` -> return `map[string]genericapiserver.PostStartHookFunc` -> do not expect the caller to know how to set up your post start hook.